numpy/core/src/umath/loops.c.src

```diff
@@ -2921,6 +2930,95 @@ OBJECT_sign(char **args, npy_intp *dimensions, npy_intp *steps, void *NPY_UNUSED
     Py_XDECREF(zero);
 }
 
+
+
+/*
+ *****************************************************************************
+ **                               FASTCLIP                                  **
+ *****************************************************************************
+ */
```

Comment:
I really think we want to split the ufunc loops into multiple files, possible one per ufunc. Keeping track of where to insert things is becoming extremely hard.

Unfortunately, right after we branch the 1.16 LTS(ish?) release is a terrible time to make the history diverge like that.

Reply:
Moved just `clip` to its own file for now

Reply:
And the macros are now separate too, in #13032